systemd/secretsd.service

```diff
@@ -1,11 +1,11 @@
 # ~/.config/systemd/user/secretsd.service
 [Unit]
-Description=Secret Storage Service
+Description=Basic Secret Storage Service
 
 [Service]
 Type=dbus
 BusName=org.freedesktop.secrets
-ExecStart=/usr/bin/env %h/.local/bin/secretsd
+ExecStart=/usr/bin/env %h/code/secretsd/secretsd.py
```

Comment:
I'm uncertain why, but the python exceptions and print statements aren't being captured by systemd, making the debugging process a little more difficult.

Reply:
I don't know about exceptions (might be the way dbus-python handles them) but regular prints to non-tty are just buffered by Python's io layer, unless print(…, flush=True) is used.

So far I hadn't added that because they're not something that was meant to be logged during normal usage in the first place. I do most debugging while having the script run from interactive terminal.